Configs/.config/waybar/modules/notifications.jsonc

```diff
@@ -1,5 +1,5 @@
     "custom/notifications": {
-        "format": "{icon} {}",
+        "format": "{} {icon}",
```

Comment:
I don't understand the change. moving the icon to the right?

Reply:
as i understand it something upstream with waybar causes the module to break and not display unless the icon is after the number.... yeah idk why, but a numerous people said this fix works so :shrug:

Reply:
can you help me find that upstream thread?

Reply:
https://github.com/Alexays/Waybar/issues/3623